src/hooks/useDomainInfo.tsx

```diff
@@ -51,6 +51,7 @@ export default function useDomainInfo({
     queryKey: ['domainInfo', { domain, antId, arioProcessId, aoNetwork }],
     queryFn: () => getDomainInfo({ domain, antId }).catch((error) => error),
     refetchOnWindowFocus: false,
+    staleTime: Infinity,
```

Comment:
this might be better as 1 hour given names can/will be reassigned/released

Reply:
When we reassign/release locally we reset this query - going to go ahead and send the to QA but will be sure to add this to the improvement to be done on this hook. Would like to break up what its doing on some of the info to separate queries.